vfcompositor/Compositor.c

```
@@ -737,44 +772,104 @@ void OnMouseMove(int x, int y, int dx, int dy) {
+        if (g_window_list_tail) g_window_list_tail->next = top_window;
+        g_window_list_tail = top_window;
+    }
+
+    // Title bar interactions
+    int rel_x = x - top_window->rect.x;
+    int rel_y = y - top_window->rect.y;
+    if (rel_y < 20) {
+        const int btn_size = 14; const int pad = 3;
+        int close_x = top_window->rect.width - pad - btn_size;
+        int min_x   = close_x - 2 - btn_size;
+        if (rel_x >= close_x && rel_x < close_x + btn_size && rel_y >= pad && rel_y < pad + btn_size) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)